google/resource_container_cluster.go

```diff
@@ -856,6 +926,22 @@ func flattenClusterNodePools(d *schema.ResourceData, config *Config, c []*contai
 	return nodePools, nil
 }
 
+func flattenMasterAuthorizedNetworksConfig(c *container.MasterAuthorizedNetworksConfig) []map[string]interface{} {
+	result := make(map[string]interface{})
+	if c.Enabled && len(c.CidrBlocks) > 0 {
+		cidrBlocks := make([]map[string]interface{}, 0, len(c.CidrBlocks))
+		for _, v := range c.CidrBlocks {
+			cidrBlocks = append(cidrBlocks, map[string]interface{}{
+				"cidr_block":   v.CidrBlock,
+				"display_name": v.DisplayName,
+			})
+		}
+		// now does this adhere to the schema?
```

Comment:
I'm not quite sure what this comment is for

Reply:
Oh, that definitely wasn't supposed to make it in.  😺